inst/report/styles.css

```diff
@@ -0,0 +1,37 @@
+.main-container {
+  max-width: 1300px;
+  margin-left: 0;
+  margin-right: 1;
+  text-align: justify;
+  text-justify: inter-word;
+}
+
+.banner {
+  background-color: #b5132c;
+  font-weight: 200;
+  position: relative;
+  padding: .5rem 1rem;
+  display: flex;
+  max-width: 1300px;
+}
+
+.logo_naomi {
+  height: 60px;
+  padding-left: 15px;
+  padding-right: 10px;
+}
+
+.logo_unaids {
+  height: 60px;
+}
+
+.text {
+  padding-left: 10px;
+  padding-top: 10px;
+  color: white;
+  font-size: 35px;
+}
+
+#translate:not(#translate[lang="t_(LANG)"]) {
```

Comment:
This means that if you want to render the report from this dir outside of package context i.e. outside of calling `generate_summary_report` that this won't work. Do you ever do that? We can work around that by having a default `styles.css` and overwriting that in `generate_summary_report`